modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -1,44 +1,32 @@
 // Module included in the following assemblies:
 //
 // * post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc
-
+ifeval::["{context}" == "creating-multi-arch-compute-nodes-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "creating-multi-arch-compute-nodes-ibm-z-lpar"]
+:ibm-z-lpar:
+endif::[]
 :_mod-docs-content-type: PROCEDURE
+// Assumption is that attribute once outside ifdef works for several level one headings.
 [id="machine-user-infra-machines-ibm-z_{context}"]
+ifdef::ibm-z[]
 = Creating {op-system} machines on {ibm-z-title} with z/VM
 
 You can create more {op-system-first} compute machines running on {ibm-z-name} with z/VM and attach them to your existing cluster.
+endif::ibm-z[]
+ifdef::ibm-z-lpar[]
+= Creating {op-system} machines on {ibm-z-title} in an LPAR
```

Comment:
Have we expanded LPAR somewhere in the docs?

Reply:
yes in the upi install docs but good idea to add it here too.